### ISSUE TITLE
Does atc24 work with mobile devices?

### ISSUE THREAD

**Issue by unoski1667** [Zedruc/ATC24-Suite#75]
If it doesn't, please add it

**Comment by thanhdanhelo** [comment]
Yes. ATC24 is PTFS, access it by using Roblox, tldr. If you're referring to ATC24 Suite, It is sort of not supported. The UI is out of the world and it's small.
![image](https://github.com/user-attachments/assets/13aacc94-7f49-4ef0-b124-f4f3e3c7e51f)

**Comment by Zedruc** [comment]
> Yes. ATC24 is PTFS, access it by using Roblox, tldr. If you're referring to ATC24 Suite, It is sort of not supported. The UI is out of the world and it's small.
> ![image](https://github.com/user-attachments/assets/13aacc94-7f49-4ef0-b124-f4f3e3c7e51f)

The UI indeed is absolute dogwater but fear not as the suite's replacement isn't too far from release 🗣
